### PR TITLE
Tracing Support for Dioxus 0.5

### LINKED ISSUE #3: Support Dioxus 0.5

### PR DESCRIPTION
**Pull request by DogeDark**
Closes #3

### CODE REVIEW

**Comment by DogeDark** [comment]
@marc2332 You down to review this?

**Comment by marc2332** [comment]
> @marc2332 You down to review this?

yeah, will review it asap!

File: README.md [hunk 1]
```diff
@@ -23,68 +23,41 @@
...
 }
 
-fn app(cx: Scope) -> Element {
-    info!("my app component just rendered!");
-    cx.render(rsx!(p { "Hello, Dioxus!" }))
+#[component]
+fn App() -> Element {
+  info!("App rendered");
+  rsx! {
+      "hi"
```
Comment (marc2332) [inline README.md]:
```suggestion
      p { "hi" }
```

**Review by marc2332 — CHANGES_REQUESTED** [review]
Hehe, just this. The rest LGTM

**Pull request merged**